### DISCUSSION TITLE
ImportError: can't import name XboxController

### DISCUSSION THREAD

**Discussion by cougarek** [pybricks/support#1739] [Q&A] [answered]
Hello community!
I keep getting below error when trying to import XboxController class. Tried this on two Windows 10 computers, Chrome and Edge browsers. Running on Lego City Hub.
Im following [this tutorial](https://pybricks.com/project/technic-42124-xbox/) and [this code](https://pybricks.com/assets/programs/set_42124_xbox.py).
```
Traceback (most recent call last):
  File "set_42124_xbox.py", line 2, in <module>
ImportError: can't import name XboxController
```

**Comment by BertLindeman** [comment, marked as answer]
The xbox controller is not compatible with the cityhub.
See the doc (press the small triangle next to compatibility):

![afbeelding](https://github.com/user-attachments/assets/99acd92d-c37b-4830-ac99-b2882040f7cc)

Bert
